### PR TITLE
use less repetitive name

### LINKED ISSUE #11689: Use less repetitive enum names for Cursor
Should be renamed to use less repetitive variants like `Cursor::Default` instead of `Cursor::DefaultCursor`.

 * Code: `components/style_traits/cursor.rs`

### PR DESCRIPTION
**Pull request by mrmiywj**
---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #11689  (github issue number if applicable).

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because just renaming

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/11696)

### CODE REVIEW

**Comment by ConnorGBrewster** [comment]
Thank you for your contribution!

It looks like you may have not tried compiling with these changes. You will need to go and fix any uses of `DefaultCursor`, `PointerCursor`, etc.. throughout the codebase.

File: components/style_traits/cursor.rs [hunk 1]
```diff
@@ -35,39 +35,39 @@ macro_rules! define_cursor {
...
-    "ew-resize" => EwResizeCursor = 26,
-    "ns-resize" => NsResizeCursor = 27,
-    "nesw-resize" => NeswResizeCursor = 28,
-    "nwse-resize" => NwseResizeCursor = 29,
-    "col-resize" => ColResizeCursor = 30,
-    "row-resize" => RowResizeCursor = 31,
-    "all-scroll" => AllScrollCursor = 32,
-    "zoom-in" => ZoomInCursor = 33,
-    "zoom-out" => ZoomOutCursor = 34,
+    "none" => No = 0,
```
Comment (KiChjang) [inline components/style_traits/cursor.rs]: None instead of No.

**Comment by ConnorGBrewster** [comment]
Looks good. Thank you for this!

@bors-servo r+
-S-awaiting-review 

<details><summary><i>Previously, highfive wrote…</i></summary>
> New code was committed to pull request.
</details>


---

Reviewed 4 of 4 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/servo/11696#-:-KJuX_C4f5hsrTGoKnrC:-1515820936)*

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * Git

**Comment by larsbergstrom** [comment]
@bors-servo retry
- #11719

**Comment by larsbergstrom** [comment]
@bors-servo retry
- #11719

**Comment by metajack** [comment]
@bors-servo retry

- #11719

**Comment by jdm** [comment]
Looks like `./mach build-cef` requires changes in ports/cef/ as well.

**Comment by ConnorGBrewster** [comment]
@bors-servo r+

-S-awaiting-review 

<details><summary><i>Previously, highfive wrote…</i></summary>
> New code was committed to pull request.
</details>


---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/servo/11696#-:-KJxWABARDfucaOPt9XF:-1586264022)*

**Comment by metajack** [comment]
```
window.rs:148:32: 148:55 error: no associated item named `AllScrollCursor` found for type `style_traits::cursor::Cursor` in the current scope
window.rs:148                 Cursor::Grab | Cursor::AllScrollCursor =>
                                             ^~~~~~~~~~~~~~~~~~~~~~~
window.rs:150:34: 150:58 error: no associated item named `NotAllowedCursor` found for type `style_traits::cursor::Cursor` in the current scope
window.rs:150                 Cursor::NoDrop | Cursor::NotAllowedCursor =>
                                               ^~~~~~~~~~~~~~~~~~~~~~~~
window.rs:155:36: 155:59 error: no associated item named `ColResizeCursor` found for type `style_traits::cursor::Cursor` in the current scope
window.rs:155                 Cursor::EwResize | Cursor::ColResizeCursor =>
                                                 ^~~~~~~~~~~~~~~~~~~~~~~
window.rs:159:36: 159:59 error: no associated item named `RowResizeCursor` found for type `style_traits::cursor::Cursor` in the current scope
window.rs:159                 Cursor::NsResize | Cursor::RowResizeCursor =>
                                                 ^~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by ConnorGBrewster** [comment]
Almost there! There are some to be fixed in `ports/geckolib/properties.mako.rs`.

<details><summary><i>Previously, highfive wrote…</i></summary>
> New code was committed to pull request.
</details>


---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/servo/11696#-:-KK1X84jYs_wpD5xwtw8:-1322883991)*

**Comment by mrmiywj** [comment]
@ConnorGBrewster Hmm, I do not find any `Cursor` in files in `ports/geckolib/`

**Comment by ConnorGBrewster** [comment]
@mrmiywj they were added in #11688. You probably need to do a rebase, let me know if you need any help with that!

**Comment by mrmiywj** [comment]
@ConnorGBrewster 
I pulled the upstream master and `git rebase master` on my branch. Then fix the code and rebase the two commits into one. I hope it's right^_^

**Comment by ConnorGBrewster** [comment]
Looks good! nice work!
@bors-servo r+

-S-awaiting-review 

<details><summary><i>Previously, mrmiywj (Ivan.Yang) wrote…</i></summary>
> @ConnorGBrewster 
> 
> I pulled the upstream master and `git rebase master` on my branch. Then fix the code and rebase the two commits into one. I hope it's right^_^
</details>


---

Reviewed 3 of 3 files at r5.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/servo/11696#-:-KK415BtHXDrzTQ9Nfe-:2011030943)*

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * Infra (git)

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * #11561

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * #11618

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * #11561

**Pull request merged**